safekeeper/src/timeline.rs

```diff
@@ -508,6 +510,27 @@ impl Timeline {
         }
     }
 
+    /// Pause timeline, forbidding writes to it, or resume after it has been paused.
+    pub async fn pause(&self, paused: bool) -> Result<()> {
```

Comment:
nit: maybe `set_paused`?  It's a bit weird to call a function called pause() to unpause something

Reply:
done